packages/blade/src/components/Form/FormHintWrapper.web.tsx

```diff
@@ -3,15 +3,7 @@ import BaseBox from '~components/Box/BaseBox';
 
 const FormHintWrapper = ({ children }: { children: React.ReactNode }): React.ReactElement => {
   return (
-    <BaseBox
-      as="span"
-      display="flex"
-      flexDirection="row"
-      alignItems="center"
-      style={{
-        wordBreak: 'break-all',
-      }}
-    >
+    <BaseBox as="span" display="flex" flexDirection="row" alignItems="flex-start" gap="spacing.2">
```

Comment:
gap wasn't there earlier. is this adding some extra spacing?

Reply:
It was there. https://github.com/razorpay/blade/blob/b9d4e13725382b07f791ad3b8bf0980e9ae64c7b/packages/blade/src/components/Form/FormHint.tsx#L90

We used spacer components before cuz RN didn't support gap prop